README.md

```diff
@@ -403,10 +404,12 @@ V1ToV2StorageMigrator.legacyStorage = legacyTokenStorage
 `CredentialBootstrap`, `CredentialDataSource`, and `Credential` contain several changes over 1.x. `Credential`s can no longer contain a null `Token`. Because of this change from 1.x, the flow for creating `Credential` without `Token`, followed by calling `Credential.storeToken` can no longer be used.
 When creating a new `Credential` with `Credential.store`, a `Token` must be provided.
 
-1.x would create a new `Credential` with null `Token` if no default `Credential` existed when calling `CredentialBootstrap.defaultCredential()`. In 2.x, `Credential.getDefaultCredential()` now returns `Credential?` instead of `Credential` if no default `Credential` exists.
+1.x would create a new `Credential` with null `Token` if no default `Credential` existed when calling `CredentialBootstrap.defaultCredential()`. In 2.x, default `Credential` can be fetched using `Credential.default` or `Credential.getDefaultAsync()`. Both of those have a type of `Credential?` instead of `Credential`, and return `null` if no default `Credential` exists.
 
 1.x contained `Credential` handling APIs in `CredentialBootstrap` and `CredentialDataSource`. All `Credential` management calls have been moved to `Credential` in 2.x. `CredentialBootstrap` has been deleted, and `CredentialDataSource` is private in 2.x.
 
+1.x provided `suspend` functions for handling creation and management of any `Credential`s. 2.x provides synchronous `Credential` management functions in addition to `suspend` functions. Note that it's still recommended to use `suspend` functions (suffixed with `Async`) if a coroutine scope is available.
```

Comment:
Is there a particular reason why using the suspend functions are preferred?

Reply:
I wrote this defensively to prevent crashes when using biometrics. But, I think the exception gets the point across, so I will remove the note